frontend/src/app/modules/games/games-saga.test.js

```diff
@@ -0,0 +1,29 @@
+import { getGames } from './games-effects'
+import { getGamesSaga } from './games-saga'
+import { call, put } from 'redux-saga/effects'
+import { GamesActions } from './games-actions'
+
+const RESPONSE = {
+  data: {},
+}
+
+describe('gameSaga', () => {
```

Comment:
fyi: I've seen many projects NOT test sagas, as they didn't see the clear benefit to it. hopefully, after your tests they will shine through

Reply:
I think it's a common misconception that tests are needed for something complex only. The main benefit that I experienced is that once you do some refactoring or renaming or automerge some nasty things can happen even in a simple code and unit tests can guard against it as they will feel it :)